File: src/main/java/com/fengsheng/handler/auto_play_tos.java
package com.fengsheng.handler;

import com.fengsheng.HumanPlayer;
import com.fengsheng.protos.Fengsheng;

public class auto_play_tos extends AbstractProtoHandler<Fengsheng.auto_play_tos> {
    @Override
    public void handle0(HumanPlayer player, Fengsheng.auto_play_tos pb) {
        player.setAutoPlay(pb.getEnable());
    }
}

File: src/main/kotlin/handler/auto_play_tos.kt
package com.fengsheng.handler

import com.fengsheng.HumanPlayer
import com.fengsheng.protos.Fengsheng

class auto_play_tos : AbstractProtoHandler<Fengsheng.auto_play_tos>() {
    public override fun handle0(r: HumanPlayer, pb: Fengsheng.auto_play_tos) {
        r.setAutoPlay(pb.enable)
    }
}